app/helpers/article_helper.rb

```diff
@@ -6,6 +6,11 @@ def article_search?
     controller_name == 'article'
   end
 
+  def article_research_starter?(document)
+    # TODO: we probably need a better place to put this...
+    document['eds_database_name'] == 'Research Starters' if document
```

Comment:
I believe @camillevilla has started putting methods like this in an `EdsDocument` module mixed into the `SolrDocument` class.  That might be a good place for this sort of thing.